extension/content.js

```diff
@@ -170,6 +170,15 @@ function addPatchDiffLinks() {
 	`);
 }
 
+function removeDiffSigns() {
+	$('.blob-code-deletion .blob-code-inner, .blob-code-addition .blob-code-inner').each((index, element) => {
+		const $element = $(element);
+		const html = $element.html();
+
+		$element.html(html.replace(/^[+-]/, ' '));
```

Comment:
No need to transform the whole HTML element. That's really inefficient. Just grab the first textNode, which is the `+`/`-` , and trim it.